app/gui/src/project-view/components/visualizations/TableVisualization/tableVizToolbar.ts

```diff
@@ -38,23 +37,33 @@ export interface FormatMenuOptions {
   textFormatterSelected: Ref<TextFormatOptions>
 }
 
+export interface ColumnNodeButton {
+  isCreateNewNodeEnabled: ToValue<boolean>
+  createNodes: (...options: NodeCreationOptions[]) => void
+  hiddenColumns: Ref<string[]>
+  vizColumnOrder: Ref<string[]>
```

Comment:
I'd use `ToValue` here--`Ref` constrains the producer of the data in a way that isn't necessary if the consumer is only reading it.